x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -748,6 +655,99 @@ async function endpointDetailsActivityLogChangedMiddleware({
   }
 }
 
+async function endpointDetailsActivityLogPagingMiddleware({
```

Comment:
Did anything change here? or you just moved it?

Reply:
Yeah, nothing changed here. Just moved it adjacent to the activity log fetch middleware method.